<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SC2</class>
 <widget class="QWidget" name="SC2">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>348</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>337</width>
    <height>348</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>337</width>
    <height>348</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Smart Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img.ico</normaloff>
    <normalon>img.ico</normalon>img.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="toolTipDuration">
   <number>-3</number>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);

</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>337</width>
     <height>348</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="3">
       <widget class="QPushButton" name="btn_division">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(247, 145, 15);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{ 
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_7">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(221, 221, 223);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="btn_left">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(247, 145, 15);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{ 
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>(</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_back">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(221, 221, 223);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>img.ico</normaloff>img.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="btn_clear">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(221, 221, 223);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="btn_power">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(247, 145, 15);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{ 
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>x²</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btn_plus">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(247, 145, 15);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{ 
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="btn_0">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>55</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>6544</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(221, 221, 223);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btn_point">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(221, 221, 223);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3" colspan="2">
       <widget class="QPushButton" name="btn_eq">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(247, 145, 15);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{ 
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_8">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(221, 221, 223);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btn_5">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(221, 221, 223);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btn_4">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(221, 221, 223);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="btn_right">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(247, 145, 15);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{ 
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_9">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(221, 221, 223);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btn_multi">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(247, 145, 15);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{ 
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btn_1">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(221, 221, 223);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btn_2">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(221, 221, 223);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btn_3">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(221, 221, 223);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btn_minus">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(247, 145, 15);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{ 
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="btn_sqrt">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(247, 145, 15);
	color: black;
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{ 
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>√</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btn_6">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>55</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(221, 221, 223);
	color: black;
	font: 15pt &quot;MS Shell Dlg 2&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color: rgb(181, 185, 188);
} /* при наведении курсора мышки */
QPushButton:pressed{
	background-color: rgb(158, 158, 158);
} /* при нажатии */</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_display">
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>25</pointsize>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);

color: rgb(38, 213, 11);</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="indent">
       <number>2</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
